src/microbe_stage/editor/CellEditorComponent.cs

```diff
@@ -1855,6 +1855,11 @@ private void OnOrganellesChanged()
 
         StartAutoEvoPrediction();
 
+        if (energyBalanceInfo != null && energyBalanceInfo.TotalProduction < energyBalanceInfo.TotalConsumption)
```

Comment:
I think it would be better to have a "PlayerEnergyBalance" event that just sends the energy balance object when it is calculated to the tutorial. Then the tutorials can flexibly decide to trigger when the ATP balance warrants an action.